payments/vouchers.go

```diff
@@ -46,3 +47,17 @@ func (v Voucher) recoverSigner() (types.Address, error) {
 	}
 	return nitroCrypto.RecoverEthereumMessageSigner(h[:], v.signature)
 }
+
+// Equal returns true if the two vouchers have the same channel id, amount and signaturesß
```

Comment:
1. Typo on this line
2. Should this (and all of the other methods) have a pointer receiver rather than a value receiver?

Reply:
We probably should be consistent and use pointer receivers. My natural instinct is to reserve pointer receivers for functions that modify the receiver but best practice is being [consistent](https://go.dev/doc/faq#methods_on_values_or_pointers:~:text=Next%20is%20consistency.%20If%20some%20of%20the%20methods%20of%20the%20type%20must%20have%20pointer%20receivers%2C%20the%20rest%20should%20too%2C%20so%20the%20method%20set%20is%20consistent%20regardless%20of%20how%20the%20type%20is%20used.%20See%20the%20section%20on%20method%20sets%20for%20details.)

Reply:
[2b78cca](https://github.com/statechannels/go-nitro/pull/824/commits/2b78ccaf18ecff12e7c66a88298a3f4409e4ea4d)